extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceHelperTest.php

```diff
@@ -363,14 +363,16 @@ public function thumbnailSizesDataProvider() {
 	 * @param $preferredWidth
 	 * @param $resultDimensions
 	 * @param $thumbnailDimensions
+	 * @param $thumbnail2xDimensions
 	 * @param $originalDimension
 	 * @dataProvider customWidthProvider
 	 */
-	public function testCustomWidthLogic( $customWidth, $preferredWidth, $resultDimensions, $thumbnailDimensions, $originalDimension ) {
+	public function testCustomWidthLogic( $customWidth, $preferredWidth, $resultDimensions, $thumbnailDimensions, $thumbnail2xDimensions, $originalDimension ) {
 		$expected = [
 			'name' => 'test',
 			'ref' => null,
 			'thumbnail' => null,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "thumbnail" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)